workshop/README.md

```diff
@@ -1,9 +1,31 @@
-# Istio 101 workshop
+# Advanced Istio course (name TBD)
```

Comment:
Again, this course is advanced compared with docker and k8s, but this is not an advanced istio course.  This is a istio 101 hands on course.

Reply:
This is just a placeholder until the formal name is approved.